examples/0605-mixed-no-watch-example.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html ng-app="demoapp">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="../bower_components/angular/angular.min.js"></script>
+    <script src="../bower_components/leaflet/dist/leaflet.js"></script>
+    <script src="../bower_components/angular-simple-logger/dist/angular-simple-logger.js"></script>
+    <script src="../dist/ui-leaflet_dev_mapped.js"></script>
+    <link rel="stylesheet" href="../bower_components/leaflet/dist/leaflet.css"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script>
+    var app = angular.module('demoapp', ['ui-leaflet']);
+    app.controller('MixedMOverlaysMarkersNestedNoWatchController', function ($scope, leafletData, $timeout, leafletLogger) {
+      leafletLogger.currentLevel = leafletLogger.LEVELS.debug;
+        var _clonedMarkers;
+        $timeout(function () {
+            //should do nothing (not watched) and only see one destroy
+            _clonedMarkers = angular.copy($scope.markers);
+        },1000);
+        $timeout(function () {
+            leafletData.getDirectiveControls().then(function (controls) {
+                //move all markers by a few decimal points
+                for (var markerName in _clonedMarkers) {
+                    var marker = _clonedMarkers[markerName];
+                    marker.lat += .05;
+                }
+                //force manual update
+                controls.markers.create(_clonedMarkers ,$scope.markers);
```

Comment:
Notice changes in examples.. IE $scope.makers is being passed in as oldMarkers. This is needed so it can figure out what to delete and update.